recipes/ghc/build.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+export LIBRARY_PATH="${PREFIX}/lib"
+export LD_LIBRARY_PATH="${PREFIX}/lib"
+export LDFLAGS="-L${PREFIX}/lib"
+export CPPFLAGS="-I${PREFIX}/include"
+ls
+if [ `uname` == Darwin ]
+then
+    export DYLD_LIBRARY_PATH="${PREFIX}/lib"
+    export STACK_ROOT="${SRC_DIR}/s"
+    ./configure --prefix=$PREFIX
+    make
+    make install
+else
+   ./configure --prefix=$PREFIX
+   make
+   make install
+fi
+#cleanup
+rm -r .stack-work
```

Comment:
What is this and why do we need to clean it?

Reply:
remnant from the recipe i used as template, removed it